src/nnet3/nnet-utils.cc

```diff
@@ -40,6 +40,106 @@ int32 NumInputNodes(const Nnet &nnet) {
   return ans;
 }
 
+int32 GetTModulusForIvector(const Nnet &nnet) {
+  std::vector<std::string> name;
+  name.push_back("ivector");
+  std::vector<int32> value;
```

Comment:
Also, TModulus violates the google style-guide rule against successive
capitals.  E.g. call it GetTimePeriodForIvector.


On Thu, Mar 10, 2016 at 9:36 PM, Vijayaditya Peddinti <
notifications@github.com> wrote:

> In src/nnet3/nnet-utils.cc
> <https://github.com/kaldi-asr/kaldi/pull/468#discussion_r55782968>:
>
> > @@ -40,6 +40,106 @@ int32 NumInputNodes(const Nnet &nnet) {
> >    return ans;
> >  }
> >
> > +int32 GetTModulusForIvector(const Nnet &nnet) {
> > +  std::vector<std::string> name;
> > +  name.push_back("ivector");
> > +  std::vector<int32> value;
>
> this variable should have a plural name. It is difficult to understand
> what values you are getting. It is better to choose a descriptive variable
> name, say periods ?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/kaldi-asr/kaldi/pull/468/files#r55782968>.
>